### ISSUE TITLE
I'm stupid. Creating Entity?!

### ISSUE THREAD

**Issue by robertamurrant** [keatontaylor/alexa-actions#87]
Hi.

I'm new to home assistant and coding and technology.

I've followed most of it to the letter, but my "This is a test" doesn't work.
The only bit I'm unsure about is the very first step. Where do I copy and paste the code into to create the  "input_text entity"

**Comment by robertamurrant** [comment]
TLDR: Restart home assistant after changing Configuration.yaml, disconnect and reconnect skill.

Got this fixed.
I tried other people's suggestions, such as removing "s" from "https://MyHomeAssistantURL, adding or removing SSL, 

Then i realised after I had originally added the creating entity code into the configuration.yaml, I hadn't restarted HA.
It didn't work after restarting HA, but then I re-set the items I had changed, and it worked. 
(another thing I changed, I disconnected the skill, and then reconnected, and didn't use a capital letter for my username [it shouldn't have one, but I think I put one in initially])
